README.md

```diff
@@ -24,9 +24,9 @@ $ rails s
 
 👤 **Omar Salem**
```

Comment:
This is not your name, it seems you cloned someone's project. Please fix this and try to understand what you're told to build

Reply:
I don't know if you're being serious but this is some quite an accusation, reusing the readme file does not make a project to be plagiarism. Just to remind you, we do a lot of collaboration work and many are the times when use the same readme file and adjust it a bit to fit our project. You see I would have appreciated it a-lot  if you had made that has optional cause it ain't part of the requirement.